Adding example for quickstart (#1796)
digital-asset/daml: docs/source/tools/visual.rst

1. Install `Graphviz <http://www.graphviz.org/download/>`_.
2. Generate a DAR from your project by running ``daml build``.
3. Generate a `dot file <https://en.wikipedia.org/wiki/DOT_(graph_description_language)>`_ from that DAR by running ``daml damlc visual <path_to_project>/dist/<project_name.dar> --dot <project_name>.dot``
4. Generate the visual graph with Graphviz by running ``dot -Tpng <project_name>.dot > <project_name>.png``

Visual for Quickstart
*********************

> 1. Generate the dar ``daml build``

Reviewer: ```suggestion
1. Generate the dar using ``daml build``
```

1. Install `Graphviz <http://www.graphviz.org/download/>`_.
2. Generate a DAR from your project by running ``daml build``.
3. Generate a `dot file <https://en.wikipedia.org/wiki/DOT_(graph_description_language)>`_ from that DAR by running ``daml damlc visual <path_to_project>/dist/<project_name.dar> --dot <project_name>.dot``

> 4. Generate the visual graph with Graphviz by running ``dot -Tpng <project_name>.dot > <project_name>.png``.

Reviewer 2: Avoid full stops after code to run like this, people often copy them by accident
```suggestion
4. Generate the visual graph with Graphviz by running ``dot -Tpng <project_name>.dot > <project_name>.png``
```

> Visual for Quickstart

Reviewer 2: I'd change this to something more descriptive, eg
```suggestion
Example: visualizing the Quickstart project
```

You'll need to update the line beneath so the title underline is long enough.

> *********************

Reviewer 2: I think you'll need a newline after this title.
```suggestion
*********************

```

> 1. Generate the dar using ``daml build``.

Reviewer 2: Can you add a sentence or two of introduction here, explaining what this is? Essentially saying if you've run through the quickstart guide (or even potentially just cloned it, that might do), you can generate a visual like this. And don't forget to link to the quickstart guide (using `:doc:` - you can search through the docs folder to see examples of how that works)

Reviewer 2: You also have an assumption that they've installed Graphviz, might want to be explicit about that.

> 1. Generate the dar using ``daml build``.

Reviewer 2: ```suggestion
1. Generate the dar using ``daml build``
```

Author: Cool thanks for the suggestion. Did not notice how often I myself do that.

> 2. Generate a dot file ``daml damlc visual dist/quickstart.dar --dot quickstart.dot``.

Reviewer 2: Avoid full stops after code to run like this, people often copy them by accident
```suggestion
2. Generate a dot file ``daml damlc visual dist/quickstart.dar --dot quickstart.dot``
```

> 3. Generate image ``dot -Tpng quickstart.dot quickstart.png``.

Reviewer 2: ```suggestion
3. Generate the visual graph with Graphviz by running ``dot -Tpng quickstart.dot quickstart.png``
```

> 

Reviewer 2: Can you add a line here to introduce the image? Just to say that running the above should produce something like this.

1. Install `Graphviz <http://www.graphviz.org/download/>`_.
2. Generate a DAR from your project by running ``daml build``.
3. Generate a `dot file <https://en.wikipedia.org/wiki/DOT_(graph_description_language)>`_ from that DAR by running ``daml damlc visual <path_to_project>/dist/<project_name.dar> --dot <project_name>.dot``
4. Generate the visual graph with Graphviz by running ``dot -Tpng <project_name>.dot > <project_name>.png``

Example: Visualizing the Quickstart project
*******************************************

> If you are working through :doc:`quickstart </getting-started/quickstart>` example you can generate a visual like this. If you do not have Graphviz installed `install it from <http://www.graphviz.org/download/>`_.

Reviewer 2: ```suggestion
Here's an example visualization based on the :doc:`quickstart </getting-started/quickstart>`. You'll need to  `install Graphviz <http://www.graphviz.org/download/>`_ to try this out.
```

1. Generate the dar using ``daml build``
2. Generate a dot file ``daml damlc visual dist/quickstart.dar --dot quickstart.dot``
3. Generate the visual graph with Graphviz by running ``dot -Tpng quickstart.dot quickstart.png``

> Running the above should produce an image which looks something like.

Reviewer 2: ```suggestion
Running the above should produce an image which looks something like this:
```
